rchicoli/docker-log-elasticsearch
# implement https and extra log-opt

- added https support for elasticsearch v5
- added new log-opts:
  - elasticsearch-sniff
  - elasticsearch-username
  - elasticsearch-password

TODO:
- add support for all elasticsearch clients
- test https via nginx
- add another elasticsearch option for tls-skip-verify
- update readme
- release new version
